data/versioned-api/transaction-handling.json

```diff
@@ -0,0 +1,232 @@
+{
+  "description": "Transaction handling",
+  "schemaVersion": "1.1",
+  "runOnRequirements": [
+    {
+      "minServerVersion": "4.7",
+      "topologies": [
+        "replicaset",
```

Comment:
This new spec test can only run against a replica set or a sharded cluster, but I still need to enable requireAPIVersion on every node to do so.